x-pack/plugins/fleet/common/types/rest_spec/fleet_setup.ts

```diff
@@ -17,4 +17,5 @@ export interface GetFleetStatusResponse {
   >;
   missing_optional_features: Array<'encrypted_saved_object_encryption_key_required'>;
   package_verification_key_id?: string;
+  is_secrets_storage_enabled?: boolean;
```

Comment:
@juliaElastic @jen-huang I wonder if in a follow up we could have a separate server side flag for output secrets (this one is for policy secrets) and display the same kind of callout in the UI when output secrets aren't available due to a missing fleet server constraint.

Reply:
Aren't the fleet server constraints the same regardless of if it's output or policy secrets? Shouldn't we be able to use the same flag?

Reply:
Not exactly, policy secrets are supported from fleet-server 8.10, output secrets from 8.12

```
export const SECRETS_MINIMUM_FLEET_SERVER_VERSION = '8.10.0';
export const OUTPUT_SECRETS_MINIMUM_FLEET_SERVER_VERSION = '8.12.0';
```